src/js/lib/detectors.js

```diff
@@ -99,8 +99,8 @@
 		var cookieName = testCookieName || 'testcookie';
 
 		if (isUndefined(navigatorAlias.cookieEnabled)) {
-			cookie.cookie(cookieName, '1');
-			return cookie.cookie(cookieName) === '1' ? '1' : '0';
+			helpers.cookie(cookieName, '1');
```

Comment:
Are we in danger of producing a warning here, because we do not set samesite on the test cookie?

(I have not yet got as far as testing this for myself - but I will do)

Reply:
I have now tested this for myself.  The test cookie is only set on very old (ancient!) browsers, for which `navigator.cookieEnabled` is undefined.  So there will be no issue with samesite warning messages on those old browsers.  So there is no issue here to worry about.